Sources/BraintreeApplePay/BTApplePayClient.swift

```diff
@@ -49,6 +52,27 @@ import BraintreeCore
         }
     }
 
+    /// Creates a `PKPaymentRequest` with values from your Braintree Apple Pay configuration.
+    /// It populates the following values of `PKPaymentRequest`: `countryCode`, `currencyCode`, `merchantIdentifier`, `supportedNetworks`.
+    /// - Returns: A `PKPaymentRequest`
+    /// - Throws: An `Error` describing the failure
```

Comment:
🤔 Stinks to have to duplicate and maintain 2 docstrings for each method now. More room for error.

Off the top of my head I don't have ideas for remedying this, but we can think on it!

Reply:
DocC might have some capabilities to allow for docstring re-use

Reply:
Yeah agree, lots of potential room for error, we can certainly look into other options though in the future!

Reply:
It's surprising that ObjC does a lot of translation out of the box - but Swift requires 2 explicit function defs.

A few options/brainstorming:
* 1 docstring
    * Only offer async-await functions. What would the conversion process look like if they need to use completion syntax? At this point, maybe most Swift merchants are prepared to use async-await.
    * Only offer completionHandler style functions. Let merchants wrap to async-await.
* 2 docstrings
    * What we have now - offering both out of the box from our SDK. This _seems_ to me like the best merchant experience. But this is assuming that some Swift merchants don't want to use async-await.
    * But also, at what cost? If our reference docs messy lost in the shuffle, this isn't worth it.
    * What internal practices and enforcements can we put in place to ensure consistency in our docstrings. More of a 🏈 punt, but worth maybe doing some high-level investigation on.

Reply:
Yeah - I think for now I would vote 2 docstring. The first go around it's a lot to maintain, but most of these were ripped right from Obj-C during the conversion process, so once they are there they don't change a ton it seems. Maybe we can start a discussion with some merchants or on GitHub discussions to see what sort of impact we'd have if we moved to just async/await in v7. May be an even higher adoption rate at that point as well that will make us ripping the completions out easier?

Reply:
> Maybe we can start a discussion with some merchants or on GitHub discussions to see what sort of impact we'd have if we moved to just async/await in v7.

Yeh for sure - I'd love to try to get some feedback on this. Though the only real way might be to remove it and see who/if anyone raises concerns.